website/content/partials/pki-double-migration-bug.mdx

```diff
@@ -0,0 +1,22 @@
+### PKI Storage Migration Revives Deleted Issuers
+
+In Vault 1.11, a new storage layout was created to support multiple issuer
+within a single mount, creating storage version 1. In Vault 1.11.6, 1.12.2,
+and 1.13.0, a bug was fixed wherein (due to incorrect ordering of writes),
+the CA chain could be incorrect, due to writes failing when too many mounts
+attempted migration at the same time; this introduced storage version 2.
+
+#### Impacted Versions
+
+If issuers were present pre-1.11, and were subsequently deleted prior to
+upgrading to a Vault version with storage version 2, the initial migration
+of issuers would be re-attempted and these issuers would be recreated. If
+managed keys were associated with these legacy versions (and were
+subsequently removed from the managed key repository), migration would fail
+with an error in the logs:
+
+> Error during migration of PKI mount:
+>   failed to lookup public key from managed key:
+>     no managed key found with uuid
+
+This has been fixed in Vault 1.14.0, 1.13.4, 1.12.8, and 1.11.12.
```

Comment:
```suggestion
```

Suggest deleting this line and making the information easier to find by highlighting it at the beginning.